src/base/bittorrent/session.cpp

```diff
@@ -3930,6 +4025,7 @@ namespace
         if (torrentData.category.isEmpty())
         // **************************************************************************************
             torrentData.category = QString::fromStdString(fast.dict_find_string_value("qBt-category"));
+        torrentData.tags = entryDictToSet(fast.dict_find_dict("qBt-tags"));
```

Comment:
Dictionary keys seemed more semantically similar to a set. Not a strong preference though. It would be fine to switch it to a list if that's what you want.